### PR TITLE
Greenkeeper is incompatible with Travis CI PR build

### LINKED ISSUE #94: Greenkeeper is incompatible with Travis-CI pull request build
**Issue**
Currently `travis.yml` specifies that if the base branch is a Greenkeeper branch that it should use `yarn install` rather than `yarn install --frozen-lockfile` however, this does not work for builds for a pull request, because the base branch in that scenario is always `master`. 

**Expected behavior**
If Greenkeeper upgrades a package on it's branch, the continuous integration should also upgrade that package when running it's build.

### PR DESCRIPTION
**Pull request by corymharper**
Currently Greenkeeper pull requests can not pass the pull request build, this pull request fixes that.

closes #94

### CODE REVIEW

**Comment by corymharper** [comment]
There appear to be multiple package versioning issues that conflict with using the newest version of Node, here's one we are experiencing:

https://github.com/babel/babel/issues/11216

I updated all of the packages as a result.

File: demo/package.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
 		"@types/react-dom": "^16.9.0",
 		"prettier": "^1.19.1",
 		"react": "^16.12.0",
-		"react-accessible-dropdown-menu-hook": "link:..",
+		"react-accessible-dropdown-menu-hook": "link:../",
```
Comment (WesCossick) [inline demo/package.json:15]: Is there a reason why this was changed?

**Review by WesCossick — COMMENTED** [review]
Left a question.

File: demo/package.json (hunk 1 above)
Comment (corymharper) [reply demo/package.json:15]: I've noticed that this changes to this every time that I run `yarn install`, maybe it should be reverted?

**Review by WesCossick — APPROVED** [review]

**Pull request merged**